<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tecnicaIndex</class>
 <widget class="QWidget" name="tecnicaIndex">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1,7">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>TÃ©cnicas</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutFiltro">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Buscar: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="le_search">
       <property name="placeholderText">
        <string>Buscar...</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bt_create">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string>Agregar nuevo</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>images/icons8-plus-24.svg</normaloff>images/icons8-plus-24.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layout_tabla"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
